source/get-started/setup/index.md

```diff
@@ -0,0 +1,203 @@
+---
+title: How To Set Up A Project
+author: ajolicoeur
+layout: page
+---
+<div class="well">
+  <p>
+    <b>HEADS UP</b><br/>
+    This document will use the command line. If you feel uncomfortable with this, you can install <a href="https://incident57.com/codekit/" target="top">CodeKit</a>. It includes Bower and will automatically reload and compile your LESS files without having to run anything in the command prompt.
+    If you are unfamiliar with command line tools, but would like to learn more, you can watch this <a href="https://www.youtube.com/playlist?list=PLLnpHn493BHGmEYzbjWPJsnRMhvs-PSYG" target="top">video series</a> to learn the basics.
+  </p>
+  <p>As part of this "how to", we are using MacOS running El Capitan.</p>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <h2>Create A Project</h2>
+    <div class="row">
+      <div class="col-xs-12 col-sm-12 col-md-6">
+        <h3>Using Finder</h3>
+        <ul>
+          <li>Launch a new Finder window</li>
+          <li>Navigate to where you want to project to live</li>
+          <li>File -> New Folder (or Ctrl + Shift + N)</li>
+          <li>Name the folder</li>
```

Comment:
The create a project section only shows how to create a directory. It may be misleading to call this a project?